docs/wiki/research/roadmap.md

```diff
@@ -17,43 +17,44 @@ The discussion, resources and all research and development on the core protocol
-| Single slot finality (SSF)           |                                           Blocks could be proposed and finalized in the same slot                                            | (i) more convenient for apps (transactions finalization time improved by an order of magnitude, i.e. 12 seconds instead of 12 minutes means better UX for all), (ii) much more difficult to attack (multi block MEV re-orgs can be eliminated and the complexity in consensus mechanism, reduced) | in research </br>(i) VB's SSF notes[^4] </br>(ii) 8192 signatures post-SSF[^5] </br>(iii) simple SSF protocol[^6] |
-| Single Secret Leader Election (SSLE) |                        Allow elected block proposers to remain private until block publishing, to prevent DoS attacks                        |                                                                                                            Only the selected validator knows it has been selected to propose a block.                                                                                                             | in research </br>EIP-7441[^7]                                                                                     |
-| Enable more Validators               | The technical challenge of efficiently coordinating an ever increasing number of validators to achieve SSF with the best trade-offs possible |                                                                                          Greater redundancy, a broader range of proposers, a wider array of attesters, and overall increased resilience                                                                                           | in research </br> (i) EIP-7514[^8] </br>(ii) EIP-7251[^9] </br> (iii) 8192 signatures[^5]                         |
-| Quantum-safe signatures              |                               Proactive research and integration of quantum-resistant cryptographic algorithms                               |                                                                                               Quantum-safe, aggregation-friendly signatures will enhance protocol security against quantum attacks                                                                                                | in research </br> (i) lattice-based[^10] </br>(ii) STARK-based [^11] systems                                      |
+| Upgrade                              | Track |          Topic           |                                                                 Description                                                                  |                                                                                                                                          Expected effect                                                                                                                                          | State of the art                                                                                                  |
```

Comment:
Since the Track column is empty in this part and later just repeats MEV, I would consider removing it. I think it's quite clear from other parts where the topic fits and it's easier to keep these tables smaller